tonic/pycompat.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+PY3 = sys.version_info[0] >= 3
+
+if PY3:  # pragma: no cover
+    basestring = str
+    unicode_type = str
+    bytes_type = bytes
+
+    def iteritems(d):
+        return iter(d.items())
+
+    def itervalues(d):
+        return iter(d.values())
+
+    pyrange = range
+    pyzip = zip
```

Comment:
pyzip is defined below in line 34: https://github.com/UW-Hydro/tonic/pull/20/files#diff-1307aa1f6fc289df0ef54e246b689727R34